src/services/parser.ts

```diff
@@ -132,7 +132,8 @@ export const getAddByRSSPodcastFeedUrlsLocally = async () => {
 
 const setAddByRSSPodcastsLocally = async (podcasts: any[]) => {
   if (Array.isArray(podcasts)) {
-    await AsyncStorage.setItem(PV.Keys.ADD_BY_RSS_PODCASTS, JSON.stringify(podcasts))
+    const cleanedPodcasts = podcasts.filter((podcast: any) => !!podcast || typeof podcast !== 'object')
```

Comment:
Is this check correct? If a podcasts exists OR its not(?) an object?

Reply:
I have no idea what I was thinking here. Now I'm thinking !! should be enough. We were seeing `null` creep its way into the subscribedPodcasts array...